tomviz/PresetModel.h

```diff
@@ -0,0 +1,38 @@
+/* This source file is part of the Tomviz project, https://tomviz.org/.
+   It is released under the 3-Clause BSD License, see "LICENSE". */
+
+#ifndef tomvizPresetModel_h
+#define tomvizPresetModel_h
+
+#include <QAbstractTableModel>
+#include <QList>
+
+#include <vtkNew.h>
+#include <vtkSMTransferFunctionPresets.h>
+
+namespace tomviz
+{
+  
+class PresetModel : public QAbstractTableModel
+{
+  Q_OBJECT
+  
+ public:
+  PresetModel(QObject *parent = nullptr);
+  int rowCount(const QModelIndex &parent = QModelIndex()) const override;
+  int columnCount(const QModelIndex &parent = QModelIndex()) const override;
+  QVariant data(const QModelIndex &index, int role = Qt::DisplayRole) const override;
+  QVariant headerData(int section, Qt::Orientation orientation, int role) const override;
+
+ signals:
+  void getPreset(const Json::Value& preset);
+  
+ public slots:
+  void handleClick(const QModelIndex &index);
+
+ private:
+  vtkNew<vtkSMTransferFunctionPresets> Presets;
```

Comment:
We usually name member variables with the `m_` prefix to better distinguish them from local variables.

Reply:
Yes, thank you! I I messed that up in more than one place, they are all corrected now.